src/scheduler/site_code.cpp

```diff
@@ -651,7 +651,7 @@ site_get_share(resource_resv *resresv)
 
 /*
  *=====================================================================
- * site_init_alloc( sinfo ) - Initialize allocated shares CPUs data
+ * site_init_allocstatic_cast< sinfo >(Initialize allocated shares CPUs data
```

Comment:
I think your find and replace failed here.

Reply:
haha, it did, thanks for noticing